devices/common_patches/fsmc/fsmc_sramfix.yaml

```diff
@@ -0,0 +1,33 @@
+"FMC,FSMC":
+  BCR1:
+    _add:
+      WRAPMOD:
+        description: WRAPMOD
+        bitOffset: 10
+        bitWidth: 1
+  "BCR[2-4]":
+    _add:
+      CCLKEN:
```

Comment:
From RM0090 and RM0386, "The CCLKEN bit of the FMC_BCR2..4 registers is don’t care. It is only enabled through the
FMC_BCR1 register.". So I guess that's why it's not in the SVD. Should we include it/does it do something useful on other devices?

Reply:
I've added some bits to make BCRx identical to collect them into array, but now I'm not sure is it really needed to collect into array.